test/e2e/e2e_suite_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	_ "github.com/triggermesh/triggermesh/test/e2e/framework"
 
 	// test suites
-	_ "github.com/triggermesh/triggermesh/test/e2e/sources"
+	// _ "github.com/triggermesh/triggermesh/test/e2e/sources"
```

Comment:
You can easily avoid this while working on a specific component by changing any `Describe / Context / When / ...` to `FDescribe / FContext / FWhen / ...`.

"F" stands for "Focus". It "focuses" a group of spec and disables all other tests.